network/vars.yaml

```diff
@@ -28,3 +28,7 @@ NEWT_SHA512: 021cb93f8616a907f5d0d2e5c2979cc28669042b2cc77322633a23e49cae48e0498
 NETBIRD_VERSION: 0.60.2
 NETBIRD_SHA256: e3ec023aff48d86ed1302708aa0522f610ec02891da8f24e6d98b9e9680bbc4e
 NETBIRD_SHA512: 8bebc0412932eb0c278749b282bae366ff032ca95c5e9032ad54e84e91b47c6f530c4f4bcf9d5843fdd4025516efdad46913f73d18976b177c2534bd7258affe
+# renovate: datasource=git-tags depName=https://gitlab.nic.cz/labs/bird.git allowedVersions=2.* versioning=semver
+BIRD2_VERSION: "2.17.2"
```

Comment:
2.17.2 doesn't exist, there is 2.17.1 only...  not sure how it came together?

Reply:
Wth, I built this yesterday on a fresh system. Gonna check tonight what's going wrong here and amend/push a fix